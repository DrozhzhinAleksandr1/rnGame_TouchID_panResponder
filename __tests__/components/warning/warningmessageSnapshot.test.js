import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { WarningMessage } from '../../../app/components/warning/warningmessage';

it('create snapshot WarningMessage', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<WarningMessage />);
    expect(toolbarSnapshot).toMatchSnapshot();
});