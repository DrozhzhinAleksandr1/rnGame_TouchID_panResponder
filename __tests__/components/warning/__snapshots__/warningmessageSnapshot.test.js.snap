// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot WarningMessage 1`] = `
<Component
  style={
    Object {
      "flex": 1,
      "height": 15,
      "justifyContent": "flex-start",
      "width": 296,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    selectTextOnFocus={false}
    style={
      Object {
        "color": "#F86F6F",
        "fontFamily": "Roboto-Regular",
        "fontSize": 11,
      }
    }
  />
</Component>
`;
